tests/acceptance/features/bootstrap/NotificationContext.php

```diff
@@ -104,6 +104,17 @@ public function userListAllNotifications(string $user):void {
 		$this->featureContext->setResponse($response);
 	}
 
+	/**
+	 * @Then the JSON response should not contain any notification
```

Comment:
Can we change it to something like this?
```suggestion
	 * @Then the notifications should be empty
```